i18n/de/docusaurus-plugin-content-docs/current/help-center/your-account/account-settings.mdx

```diff
@@ -26,17 +26,19 @@ Ein Profilbild zu haben ist nützlich, da es anderen Benutzern ermöglicht, Sie
   <TabItem value="desktop" label="Desktop" default>
   <ol>
     <li>Klicken Sie auf Ihr Profilbild oder Ihre Initialen, um die Einstellungen zu öffnen.</li>
+    <li>Klicken Sie auf <b>Profil</b>.</li>
     <li>Klicken Sie auf Ihr Profilbild oder Ihre Initialen.</li>
-    <li>Klicken Sie auf "Galerie eröffnen".</li>
+    <li>Klicken Sie auf <b>Galerie öffnen</b>.</li>
     <li>Wählen Sie ein Bild aus Ihren Dateien.</li>
   </ol>
   </TabItem>
   <TabItem value="mobil" label="Mobil">
   <ol>
     <li>Tippen Sie auf Ihr Profilbild oder Ihre Initialen, um die Einstellungen zu öffnen.</li>
+    <li>Tippen Sie auf <b>Profil</b>.</li>
     <li>Tippen Sie auf Ihr Profilbild oder Ihre Initialen.</li>
-    <li>Tippen Sie auf "Galerie eröffnen" or "Kamera eröffnen".</li>
-    <li>Wählen Sie ein Bild aus Ihren Dateien.</li>
+    <li>Tippen Sie auf <b>Galerie eröffnen</b> oder <b>Kamera öffnen</b>.</li>
+    <li>Wählen Sie ein Bild aus Ihren Dateien, oder schiessen Sie ein neues mit der Kamera.</li>
```

Comment:
Ein Foto "schießen" sagt zumindest in Deutschland so gut wie niemand, auch wenn es nicht falsch ist.
Ich fände "oder nehmen Sie ein Neues mit der Kamera auf" besser.
Vor ein "oder" gehört außerdem kein Komma